tests/test_dev_server.py

```diff
@@ -2,38 +2,37 @@
 import io
 import socket
 from socketserver import BaseServer
-from unittest import mock
+from unittest.mock import patch, MagicMock, call
 
-from lbz.response import Response
-from lbz.resource import Resource
-from lbz.router import add_route
+import pytest
 
-from lbz.dev.server import MyDevServer
-from lbz.dev.server import MyLambdaDevHandler
+from lbz.dev.server import MyDevServer, MyLambdaDevHandler
+from tests.sample_test_resources import SimpleTestResource
 
+# TODO: Check why in some tests there are two responses - its not affecting actual runtime
 
-class HelloWorld(Resource):
-    @add_route("/", method="GET")
-    def list(self):
-        return Response({"message": "HelloWorld"})
 
-    @add_route("/t/{id}", method="GET")
-    def get(self):
-        return Response({"message": "HelloWorld"})
+def test_my_dev_server_no_cls_raises() -> None:
+    with pytest.raises(TypeError):
+        MyLambdaDevHandler.cls()  # pylint: disable=no-value-for-parameter
 
 
 class MyLambdaDevHandlerHelloWorld(MyLambdaDevHandler):
-    cls = HelloWorld
+    cls = SimpleTestResource
```

Comment:
I would keep that with the `HelloWorld` as it was implemented before. The more complicated code written only for tests makes tests more complicated - keep it simple 😉  Most of the things that are implemented inside this resource should be unit tested anyway.